bin/server_package/installation/default

```diff
@@ -17,3 +17,6 @@ MYSTERIUM_DISCOVERY_ADDRESS: ""
 
 # Mysterium broker address (eg.: testnet.mysterium.network)
 MYSTERIUM_BROKER_ADDRESS: ""
+
+# Mysterium server protocol (options: {udp, tcp})
+MYSTERIUM_SERVER_PROTOCOL: ""
```

Comment:
Wrong syntax here `VAR: "value"` should be `VAR="value"`

Reply:
But @tadovas is dropping container ENV variables, dont spent time on introducing new ones